localized/en-us/games/wordwindow/wordwindow_level.tsv

```diff
@@ -187,55 +187,55 @@ en-US	14	1	5	4	"बच्चे अपनी कक्षा को सजान
 en-US	14	2	1	4	"अन्ना ने १२ पौधे खरीदे। उसकी बहन ने ७ पौधे खरीदे। अन्ना ने अपनी बहन से कितने अधिक पौधे खरीदे?"	¥óææ Ùð 12 ÂõÏð ¹ÚèÎðÐ ©â·UUUè ÕãÙ Ùð 7 ÂõÏð ¹ÚèÎðÐ ¥óææ Ùð ¥ÂÙè ÕãÙ âð ç·UUUUÌÙð ¥çÏ·UUUU ÂõÏð ¹ÚèÎð?	3	5	7	12	plants	plants	plants	plants	12 - 7 = <5>	2	12	7	0	0					wordwindow-stone-yellow.png	wordwindow-stone-blue.png		
 en-US	14	2	2	4	"मार्च में १५ दिन बारिश हुई। अप्रैल में ८ दिन बारिश हुई। मार्च में अप्रैल की तुलना में कितने दिन अधिक बारिश हुई?"	×æ¿ü ×ð¢ 15 çÎÙ ÕæçÚàæ ãé§üÐ ¥ÂýñÜ ×ð¢ 8 çÎÙ ÕæçÚàæ ãé§üÐ ×æ¿ü ×ð¢ ¥ÂýñÜ ·UUUè ÌéÜÙæ ×ð¢ ç·UUUUÌÙð çÎÙ ¥çÏ·UUUU ÕæçÚàæ ãé§ü?	5	7	8	15	days	days	days	days	15 - 8 = <7>	2	15	8	0	0					wordwindow-stone-blue.png	wordwindow-stone-yellow.png		
 en-US	14	2	3	4	"फ्रेड १२ साल का है। उसका भाई टॉम ८ साल का है। फ्रेड की तुलना में टॉम कितना छोटा है?"	ÈýðUUUÇ 12 âæÜ ·UUUæ ãñÐ ©â·UUUæ Öæ§ü Åæò× 8 âæÜ ·UUUæ ãñÐ ÈýðUUUÇ ·UUUè ÌéÜÙæ ×ð¢ Åæò× ç·UUUUÌÙæ ÀôÅæ ãñ?	4	8	12	18	years	years	years	years	12 - 8  = <4>	1	12	8	0	0					wordwindow-stone-green.png	wordwindow-stone-red.png		
-en-US	14	2	4	4	"जेन १५ कुकीज़ और ७ केक बेक करती है। कुकीज़ की तुलना में उसने कितने कम केक बेक किए?"	ÁðÙ 15 ·éUUU·UUUèÁ¸ ¥æñÚ 7 ·ðUUU·UUUU Õð·UUUU ·UUUUÚÌè ãñÐ ·éUUU·UUUèÁ¸ ·UUUè ÌéÜÙæ ×ð¢ ©âÙð ç·UUUUÌÙð ·UUUU× ·ðUUU·UUUU Õð·UUUU ç·UUUU°?	7	8	9	10	cakes	cakes	cakes	cakes	15 - 7 = <8>	2	15	7	0	0					wordwindow-stone-red.png	wordwindow-stone-green.png		
+en-US	14	2	4	4	"जेन १५ बिस्कुट और ७ केक बेक करती है। बिस्कुट की तुलना में उसने कितने कम केक बेक किए?"	ÁðÙ 15 çÕS·¤éÅU ¥õÚU 7 ·Ô¤·¤ Õð·¤ ·¤ÚUÌè ãñÐ çÕS·¤éÅU ·¤è ÌéÜÙæ ×ð´ ©âÙð ç·¤ÌÙð ·¤× ·Ô¤·¤ Õð·¤ ç·¤°??	7	8	9	10	cakes	cakes	cakes	cakes	15 - 7 = <8>	2	15	7	0	0					wordwindow-stone-red.png	wordwindow-stone-green.png		
```

Comment:
Remove extra question mark from Chanakya text.
Mistake in Chanakya text: अौर

Reply:
Updated chanakya text and removed question mark